app.js

```diff
@@ -23,15 +27,55 @@ app.use(bodyParser.json());
 
 app.use(helmet());
 
-app.post('/signin', login);
-app.post('/signup', createUser);
+app.use(requestLogger);
+app.use(cookieParser());
+
+// удалить поздже
+app.get('/crash-test', () => {
+  setTimeout(() => {
+    throw new Error('Сервер сейчас упадёт');
+  }, 0);
+});
+
+app.post('/signin', celebrate({
+  body: Joi.object().keys({
+    email: Joi.string().required().email(),
+    password: Joi.string().required().min(5),
+  }),
+}), login);
+
+app.post('/signup', celebrate({
+  body: Joi.object().keys({
+    name: Joi.string().required().min(2).max(30),
+    about: Joi.string().required().min(2).max(30),
+    avatar: Joi.string().required().uri({ allowRelative: true }),
```

Comment:
Можно лучше: убрать `allowRelative` в `Joi.uri()`.
```suggestion
    avatar: Joi.string().required().uri(),
```